test/other/codesize/test_codesize_cxx_ctors1.size

```diff
@@ -1 +1 @@
-129166
+129206
```

Comment:
Seems like a 40 byte increase is more than I would expect from this change.. I was hoping for more like 6 bytes (2 bytes each for the 3 extra store instructions).

Reply:
Well we also added `pthread_getattr_np.c` which looks like:
```C
int pthread_getattr_np(pthread_t t, pthread_attr_t *a)
{
	*a = (pthread_attr_t){0};
	a->_a_detach = t->detach_state>=DT_DETACHED;
	a->_a_guardsize = t->guard_size;
	a->_a_stackaddr = (uintptr_t)t->stack;
	a->_a_stacksize = t->stack_size;
	return 0;
}
```
Presumably the other 34 bytes come from that?

Reply:
But that file should not linking into any of those tests.  It should only be linked into the (rare) programs that use it.